WeCamp/flyingliquourice
# Add virtual environment to the project

### What has been done
- Updated composer.lock to updated refs
- Added Vagrantfile to create a virtual/dev environment
- Added Ansible to setup our virtual/dev environment
- Make the daemon run independently of directory
### How to test
- Make sure you have Vagrant, Virtualbox and Ansible installed
- To allow automatic changes to your hostfile als add the vagrant plugin vagrant-hostsupdater
